_patterns/footer/footer.css

```diff
@@ -50,18 +50,18 @@
 
 /* Includes for black text and link content */
 .nav-black a {
-  color: var(--color-primary-dark);
+  color: var(--color-secondary-dark);
```

Comment:
This reverts a colour fix I made earlier, but since we'll be rebuilding the hover states with box shadows later, it's not a problem :)